docs/en/sql-reference/statements/grant.md

```diff
@@ -498,6 +505,23 @@ and
 [`format_display_secrets_in_show_and_select` format setting](../../operations/settings/formats#format_display_secrets_in_show_and_select)
 are turned on.
 
+### NAMED COLLECTION ADMIN
```

Comment:
Worth to add a note tha before 23.7 it was called NAMED_COLLECTION_CONTROL, and after 23.7 it is preserved as an alias.